### PR TITLE
Small update on security concept page

### PR DESCRIPTION
**Pull request by myidpt**
Please provide a description for what this PR is for.

Updating the page: https://preliminary.istio.io/latest/docs/concepts/security/

This PR also resizes the pictures that seem to be too large in the page.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ X ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/concepts/security/index.md [hunk 1]
```diff
@@ -118,13 +116,13 @@ work together with `istiod` to automate key and certificate
...
-{{< image width="75%"
+{{< image width="40%"
     link="./id-prov.svg"
-    caption="Identity Provision"
+    caption="Identity Provisioning Workflow"
     >}}
 
-Istio provisions identities through the
-[secret discovery service (SDS)](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret#secret-discovery-service-sds)
+Istio Envoy provisions keys and certificates through the
```
Comment (ericvn) [inline content/en/docs/concepts/security/index.md:124]: Maybe @frankbu has a thought, but I'm not sure `Istio Envoy` makes sense here. I'm thinking the original `Istio` is better.

Comment (frankbu) [reply content/en/docs/concepts/security/index.md:124]: I agree with @ericvn 
```suggestion
Istio provisions keys and certificates through the
```

File: content/en/docs/concepts/security/index.md [hunk 2]
```diff
@@ -118,13 +116,13 @@ work together with `istiod` to automate key and certificate
...
+{{< image width="40%"
     link="./id-prov.svg"
-    caption="Identity Provision"
+    caption="Identity Provisioning Workflow"
     >}}
 
-Istio provisions identities through the
-[secret discovery service (SDS)](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret#secret-discovery-service-sds)
+Istio provisions keys and certificates through the
+[Envoy secret discovery service (SDS)](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret#secret-discovery-service-sds)
```
Comment (liminw) [inline content/en/docs/concepts/security/index.md:125]: Do we need to mention Envoy? Or SDS is enough?

Comment (frankbu) [reply content/en/docs/concepts/security/index.md:125]: Mentioning Envoy here seems good.

**Review by frankbu — APPROVED** [review]

**Pull request merged**